### PR TITLE
Add new article

### PR DESCRIPTION
**Pull request by ghost**
Ready for proof-reading and critique. Thanks!

### CODE REVIEW

File: lib/helpers/multiblog.rb [hunk 1]
```diff
@@ -123,6 +125,7 @@ def contact_tag_for(item)
         # Guest bloggers
         case author
         when "Marko Wenzel", "Erik Pfeiffer" then return ""
+        when "Nick" then return %Q{&bull; <a href="https://github.com/xHN35RQ">Github</a>}
```
Comment (DivineDominion) [inline lib/helpers/multiblog.rb:128]: :D Good work on making yourself part of this without me interfering :)

File: content/posts/2016/12/trello-zettelkasten-implementation.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+### More Features
+
+See [How to use Trello like a pro](http://help.trello.com/article/734-how-to-use-trello-like-a-pro) and the [Trello Resources Board](https://trello.com/b/nPNSBZjB/trello-resources) for more useful ways to work with your lists and cards.
+
+## Conclusion
+
+Trello offers a flexible set of tools to build a knowledge base. It is especially suited for collaborative note archives. However, it lacks a generic data import tool. Also, if you lose internet access you will lose access to your note archives. For these reasons, it seems Trello is best suited for small or new note archives, project-based note archives, and for people who have a constant internet connection.
+
+Let us know if you [try Trello](https://trello.com/zettel/recommend) to build a Zettel note archive. And tell us if you have any questions or suggestions to improve this article.
```
Comment (DivineDominion) [inline content/posts/2016/12/trello-zettelkasten-implementation.md]: Is this an affiliate link? If so, what's the benefit for you if people sign up through this link? (Please add a short notice for transparency)

**Comment by DivineDominion** [comment]
The article is a very comprehensive technical documentation, thanks Nick! I love that you employ so many GIFs, too 👍 

Is the S3 bucket where the images are hosted yours? You can add them to the blog directly (and not worry about keeping them online yourself). Just create a subfolder called `content/posts/2016/12/trello-zettelkasten-implementation/` and then you can use the `insert_rel_image` helper to reference the image file by filename and have nanoc expand the whole path.

Next to the mostly technical conclusion, I'd love to know more about your ... feelings :) How was it using the app? Did you _experience_ bottlenecks and would you personally prefer some feature change to make it work better/at all?

File: lib/helpers/multiblog.rb (hunk 1 above)
Comment (ghost) [reply lib/helpers/multiblog.rb:128]: thanks

File: content/posts/2016/12/trello-zettelkasten-implementation.md (hunk 2 above)
Comment (ghost) [reply content/posts/2016/12/trello-zettelkasten-implementation.md]: It gives Trello Gold to the account when people use it. however I'll just remove it entirely.

**Comment by ghost** [comment]
Thanks!

The images are attachments on cards on the Trello version of this article on this board: https://trello.com/b/Mgo52VBw so the images serve the dual purpose of illustrating the Trello board, and this article. I won't close this board or the associated Trello account, so the images will remain available.

I tried adding the images directly into the blog, and it worked well. However nanoc scaled each image up to 1500 pixels wide... including the GIF's. Which resulted in GIF files several megabytes in size. I wasn't sure why this is happening, and since I had already uploaded the images to Trello, I just linked them.

Is there a way to place images into the article without nanoc processing them?

Sure, I'll add some more thoughts to the conclusion. Would you like to see more 'feelings' :smile: throughout the article or mainly within the conclusion?

File: content/posts/2016/12/trello-zettelkasten-implementation.md (hunk 2 above)
Comment (ghost) [reply content/posts/2016/12/trello-zettelkasten-implementation.md]: fixed 96f2647cf0dfa9c63276d830bf4f2cd2f55eca60

**Comment by DivineDominion** [comment]
I removed the thumbnailization of post attachments. GIFs should now just work as expected.

If you want, bring in your voice wherever you feel like it. I suggested you expand the conclusion so you don't have to touch too many places. Whatever suits you best. 👍

**Comment by ghost** [comment]
Cool, I'll test it out once you push up the changes, as I'd rather have the images in-repo than hotlinked. Currently expanding the conclusion and will add some more thoughts at relevant points throughout the article.

**Comment by DivineDominion** [comment]
Forgot to push, but it's up now :)

**Comment by ghost** [comment]
Ahhh, perfect! Works fine. And the site build times are down as well. :wink:

**Comment by ghost** [comment]
With c3c2e5866144357b4d453b99974203a2529d66c1 I have expanded the conclusion and edited the article to include some perspective in relevant locations. I think it's more or less ready to publish.

**Comment by xHN35RQ** [comment]
Oh, I also re-based so the article is a single commit.

**Comment by DivineDominion** [comment]
Sorry for the delay -- instead of holidays, I had illnessdays, and wasn't very productive at all. Getting up to speed now.

I like the new version! Since someone decided to squash changes, I couldn't diff the updates (:p), but reading it as a whole again made me find out new things I liked and would have follow-up questions about:

* [Creating Cards by Email](https://github.com/xHN35RQ/zettelkasten.de/blob/c3c2e5866144357b4d453b99974203a2529d66c1/content/posts/2016/12/trello-zettelkasten-implementation.md#creating-cards-by-email): did this seem to be a killer feature? -- Some people want to have their archive web-based and writable from anywhere. I think it is only a very minor inconvenience if you need to go to your desktop computer to write a proper Zettel, but well, people are people :) 
* [Card Links](https://github.com/xHN35RQ/zettelkasten.de/blob/c3c2e5866144357b4d453b99974203a2529d66c1/content/posts/2016/12/trello-zettelkasten-implementation.md#card-links): I absolutely love how carefully crafted your [Table of Contents card](https://trello.com/c/LxIMsZlz) appears to be. Would be a great place to link people to it again and rave a bit about the nested list + link structure if you want.
* Importing: there are [IFTTT](https://ifttt.com/trello) recipes that may help people connect their feed reader to a Zettel inbox or something similar. Dunno if that's something you want to cover; just want to make sure you know about that, too!

If you don't want to make any changes, I'll merge it.

**Comment by ghost** [comment]
Ah, sorry to hear that. Hope you're feeling better soon. And my squash was an evil plan to get you to re-read the whole article. Guess it worked!

You raise some good points. Creating cards via email and IFTTT are killer features. And the Table of Contents is just a list of links, but I'll demonstrate how easy it was to make. IFTTT doesn't really allow you to mass-import cards, so I don't view it as an import tool, just another way to create cards.

**Comment by ghost** [comment]
I updated in response to the points you made, and this time no squashing. :) If everything looks good, please publish. Thanks!

**Comment by ghost** [comment]
after that final edit... LGTM 👍

**Comment by DivineDominion** [comment]
Great! 👍 I'll give Sascha another chance to read and comment & then will merge the post. (And adjust the date if needed.)

**Comment by DivineDominion** [comment]
Aaaaaaand we're live!

http://zettelkasten.de/posts/trello-zettelkasten-implementation/

Thanks again for putting in all the work!

**Pull request merged**